config/wbstack.php

```diff
@@ -27,4 +27,5 @@
 
     'qs_batch_pending_timeout' => env('WBSTACK_QS_BATCH_PENDING_TIMEOUT', 'PT300S'),
     'qs_batch_mark_failed_after' => intval(env('WBSTACK_QS_BATCH_MARK_FAILED_AFTER', '3')),
+    'qs_batch_entity_limit' => intval(env('WBSTACK_QS_BATCH_ENTITY_LIMIT', '10')),
```

Comment:
Theoretically, this would warrant a chart update, but maybe we can defer this until we actually need another value?

Reply:
we can do this whenever given there is a default :)